test/views/vat/AgriculturalFlatRateSchemeViewSpec.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.vat
+
+import play.api.data.Form
+import forms.vat.AgriculturalFlatRateSchemeFormProvider
+import models.vat.AgriculturalFlatRateScheme
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.vat.agriculturalFlatRateScheme
+
+class AgriculturalFlatRateSchemeViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "agriculturalFlatRateScheme"
+
+  val form = new AgriculturalFlatRateSchemeFormProvider()()
+
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView = () => agriculturalFlatRateScheme(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => agriculturalFlatRateScheme(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def viewIncludes(s: String): Unit = asDocument(createView()).text() must include(s)
```

Comment:
Consider removing this method which is duplicated across the ViewSpecs and using _assertContainsText_ method which is part of _ViewSpecBase_

Reply:
Done